charts/elasticsearch-curator/templates/cronjob.yml

```diff
@@ -0,0 +1,51 @@
+apiVersion: batch/v2alpha1
+kind: CronJob
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+spec:
+  schedule: "{{ .Values.schedule }}"
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            app: {{ template "fullname" . }}
+          annotations:
+            checksum/secrets: {{ include (print $.Template.BasePath "/secrets.yaml") . | sha256sum }}
```

Comment:
This is still checksumming the `secrets.yaml` before the values are interpolated right? I suppose it's better than nothing.

It's something to keep in mind, change just a value in the secrets and upgrading the helm chart would not re-create the pod with the new environment variables with the correct values unfortunately.